tuna/miopen/worker/fin_class.py

```diff
@@ -706,6 +723,7 @@ def __update_fdb_w_kernels(self,
         if fdb_obj[check_str]:
           #returned entry is added to the table
           fdb_entry = self.__compose_fdb_entry(session, fin_json, fdb_obj)
+          self.__check_layout_mismatch(fdb_entry, slv_stat)
```

Comment:
so if the layout doesnt match, this wont kill the tuning job it will just set the error status for the job. Should we at least print some warning?

Reply:
Yes, this will set the job state to errored with an appropriate result message. There's no need to stop the tuning overall.